core/tests.py

```diff
@@ -0,0 +1,15 @@
+# coding: utf-8
+from django.test import TestCase
+
+
+class AboutpageTest(TestCase):
+    def setUp(self):
+        self.resp = self.client.get('/about/')
+
+    def test_get(self):
+        'GET / must return status code 200'
```

Comment:
Exposing the result of the test is something new to me.
In my honest opinion the function name (test name) could be better.
I like to think that function names should describe the reason they exist. Lots of function will perform a `GET` methods but only this one will check if `get a page that should return 200 OK`.